harper-core/src/linting/spelled_numbers.rs

```diff
@@ -14,14 +14,14 @@ impl Linter for SpelledNumbers {
             let (number, _suffix) = number_tok.kind.number().unwrap();
             let number: f64 = number.into();
 
-            if (number - number.floor()).abs() < f64::EPSILON && number <= 10. {
+            if (number - number.floor()).abs() < f64::EPSILON && number < 10. {
                 lints.push(Lint {
                     span: number_tok.span,
                     lint_kind: LintKind::Readability,
                     suggestions: vec![Suggestion::ReplaceWith(
                         spell_out_number(number as u64).unwrap().chars().collect(),
                     )],
-                    message: "Try to spell out numbers less than a hundred.".to_string(),
+                    message: "Try to spell out numbers less than ten.".to_string(),
```

Comment:
It is kinda funny that this is spelled out when under my change it shouldn't be.

I've always worked on the principle of:
- Spelled out for < 10
- Numerals for > 10
- Whatever feels best for 10 in the moment